### PR TITLE
Favicon updates

### PR DESCRIPTION
**Pull request by vin-ni**
Updated favicon color, sizes and og image

closes https://github.com/Gencaster/gencaster/issues/419

### CODE REVIEW

File: caster-editor/public/ogimage.jpg
Comment (capital-G) [inline caster-editor/public/ogimage.jpg:1]: jpg?

File: caster-front/src/public/ogimage.jpg
Comment (capital-G) [inline caster-front/src/public/ogimage.jpg:1]: why do we have `caster-front/src/public/ogimage.jpg` and `caster-front/public/ogimage.jpg`?
Wouldn't one be sufficient here?

**Review by capital-G — CHANGES_REQUESTED** [review]
Have you setup pre-commit properly? I would guess not, because otherwise the commit would have been fixed :)

`front` has two times the same `ogimage`, and is it right that these are jpg? Seems like `png` would be more fitting here and https://ogp.me/ does not specify a file format.

File: caster-editor/public/ogimage.jpg
Comment (vin-ni) [reply caster-editor/public/ogimage.jpg:1]: let's go with pngs

**Pull request merged**